File: src/main/java/com/zw/cf/service/FileService.java
package com.zw.cf.service;

import com.zw.plug.Response;

import java.io.InputStream;

/**
 * Created by zhaowei on 2017/8/17.
 */
public interface FileService {
    Response add(InputStream file,String id, String corporationid);
    Response upToken();
    Response delete(String key);
}

File: frontend/src/app/common/restService/FileService.ts
/**
 * Created by zhaowei on 2017/10/16.
 */
import {Injectable} from '@angular/core';
import {HttpServer} from '../service/HttpServer';
@Injectable()
export class FileService {

  url = './cfmy/file/:params1/:params2/:params3/:params4/:params5';
  urls = {
    upToken: {
      method: 'get',
      params: {
        params1: 'upToken'
      }
    }
  };
  constructor( httpServer: HttpServer) {
    httpServer.S(this);
  }
}
